collector_test.go

```diff
@@ -9,6 +9,72 @@ import (
 	"github.com/prometheus/snmp_exporter/config"
 )
 
+func TestPduToSample(t *testing.T) {
+	cases := []struct {
+		pdu            *gosnmp.SnmpPDU
+		indexOids      []int
+		metric         *config.Metric
+		oidToPdu       map[string]gosnmp.SnmpPDU
+		expectedResult string
+	}{
+		{
+			pdu: &gosnmp.SnmpPDU{
+				Name:  "1.3.6.1.2.1.31.1.1.1.11.629",
```

Comment:
Use dummy oids for testing, otherwise you have to make sure the data makes sense for this oid to have